test failure in Ubuntu armhf "KeyError: dtype('int32')"

Hello, looks like a new test always failed on armhf in Ubuntu (probably also Debian)

```
Removing autopkgtest-satdep (0) ...
autopkgtest [10:03:18]: test command1: set -e ; cp -r test "$AUTOPKGTEST_TMP" ; for py in $(py3versions -r 2>/dev/null) ; do cd "$AUTOPKGTEST_TMP" ; echo "Testing with $py:" ; $py -m pytest -v test ; done
autopkgtest [10:03:18]: test command1: [-----------------------
Testing with python3.8:
============================= test session starts ==============================
platform linux -- Python 3.8.0+, pytest-4.6.6, py-1.8.0, pluggy-0.13.0 -- /usr/bin/python3.8
cachedir: .pytest_cache
rootdir: /tmp/autopkgtest.B6xCOp/autopkgtest_tmp
collecting ... collected 446 items

test/test_abaqus.py::test[mesh0] PASSED                                  [  0%]
test/test_abaqus.py::test[mesh1] PASSED                                  [  0%]
test/test_abaqus.py::test[mesh2] PASSED                                  [  0%]
test/test_abaqus.py::test[mesh3] PASSED                                  [  0%]
test/test_abaqus.py::test[mesh4] PASSED                                  [  1%]
test/test_abaqus.py::test[mesh5] PASSED                                  [  1%]
test/test_abaqus.py::test[mesh6] PASSED                                  [  1%]
test/test_abaqus.py::test[mesh7] PASSED                                  [  1%]
test/test_abaqus.py::test[mesh8] PASSED                                  [  2%]
test/test_abaqus.py::test_reference_file[False-UUea.inp-4950.0-50] PASSED [  2%]
test/test_abaqus.py::test_reference_file[True-UUea.inp-4950.0-50] PASSED [  2%]
test/test_ansys.py::test[False-mesh0] PASSED                             [  2%]
test/test_ansys.py::test[False-mesh1] PASSED                             [  2%]
test/test_ansys.py::test[False-mesh2] PASSED                             [  3%]
test/test_ansys.py::test[False-mesh3] PASSED                             [  3%]
test/test_ansys.py::test[False-mesh4] PASSED                             [  3%]
test/test_ansys.py::test[False-mesh5] PASSED                             [  3%]
test/test_ansys.py::test[True-mesh0] PASSED                              [  4%]
test/test_ansys.py::test[True-mesh1] PASSED                              [  4%]
test/test_ansys.py::test[True-mesh2] PASSED                              [  4%]
test/test_ansys.py::test[True-mesh3] PASSED                              [  4%]
test/test_ansys.py::test[True-mesh4] PASSED                              [  4%]
test/test_ansys.py::test[True-mesh5] PASSED                              [  5%]
test/test_cgns.py::test[mesh0] PASSED                                    [  5%]
test/test_cli.py::test_cli PASSED                                        [  5%]
test/test_dolfin.py::test_dolfin[mesh0] PASSED                           [  5%]
test/test_dolfin.py::test_dolfin[mesh1] PASSED                           [  6%]
test/test_dolfin.py::test_dolfin[mesh2] PASSED                           [  6%]
test/test_dolfin.py::test_dolfin[mesh3] PASSED                           [  6%]
test/test_dolfin.py::test_dolfin[mesh4] PASSED                           [  6%]
test/test_dolfin.py::test_generic_io PASSED                              [  6%]
test/test_exodus.py::test_io[mesh0] PASSED                               [  7%]
test/test_exodus.py::test_io[mesh1] PASSED                               [  7%]
test/test_exodus.py::test_io[mesh2] PASSED                               [  7%]
test/test_exodus.py::test_io[mesh3] PASSED                               [  7%]
test/test_exodus.py::test_io[mesh4] PASSED                               [  8%]
test/test_exodus.py::test_io[mesh5] PASSED                               [  8%]
test/test_exodus.py::test_io[mesh6] PASSED                               [  8%]
test/test_exodus.py::test_io[mesh7] PASSED                               [  8%]
test/test_exodus.py::test_io[mesh8] PASSED                               [  8%]
test/test_exodus.py::test_io[mesh9] PASSED                               [  9%]
test/test_exodus.py::test_io[mesh10] PASSED                              [  9%]
test/test_exodus.py::test_io[mesh11] PASSED                              [  9%]
test/test_exodus.py::test_io[mesh12] PASSED                              [  9%]
test/test_exodus.py::test_io[mesh13] PASSED                              [ 10%]
test/test_exodus.py::test_io[mesh14] PASSED                              [ 10%]
test/test_exodus.py::test_generic_io PASSED                              [ 10%]
test/test_flac3d.py::test[mesh0] PASSED                                  [ 10%]
test/test_flac3d.py::test[mesh1] PASSED                                  [ 10%]
test/test_flac3d.py::test_reference_file PASSED                          [ 11%]
test/test_gmsh.py::test_gmsh2[False-mesh0] PASSED                        [ 11%]
test/test_gmsh.py::test_gmsh2[False-mesh1] PASSED                        [ 11%]
test/test_gmsh.py::test_gmsh2[False-mesh2] PASSED                        [ 11%]
test/test_gmsh.py::test_gmsh2[False-mesh3] PASSED                        [ 12%]
test/test_gmsh.py::test_gmsh2[False-mesh4] PASSED                        [ 12%]
test/test_gmsh.py::test_gmsh2[False-mesh5] PASSED                        [ 12%]
test/test_gmsh.py::test_gmsh2[False-mesh6] PASSED                        [ 12%]
test/test_gmsh.py::test_gmsh2[False-mesh7] PASSED                        [ 13%]
test/test_gmsh.py::test_gmsh2[False-mesh8] PASSED                        [ 13%]
test/test_gmsh.py::test_gmsh2[False-mesh9] PASSED                        [ 13%]
test/test_gmsh.py::test_gmsh2[False-mesh10] PASSED                       [ 13%]
test/test_gmsh.py::test_gmsh2[False-mesh11] PASSED                       [ 13%]
test/test_gmsh.py::test_gmsh2[False-mesh12] PASSED                       [ 14%]
test/test_gmsh.py::test_gmsh2[False-mesh13] PASSED                       [ 14%]
test/test_gmsh.py::test_gmsh2[False-mesh14] PASSED                       [ 14%]
test/test_gmsh.py::test_gmsh2[False-mesh15] PASSED                       [ 14%]
test/test_gmsh.py::test_gmsh2[False-mesh16] PASSED                       [ 15%]
test/test_gmsh.py::test_gmsh2[False-mesh17] PASSED                       [ 15%]
test/test_gmsh.py::test_gmsh2[False-mesh18] PASSED                       [ 15%]
test/test_gmsh.py::test_gmsh2[True-mesh0] PASSED                         [ 15%]
test/test_gmsh.py::test_gmsh2[True-mesh1] PASSED                         [ 15%]
test/test_gmsh.py::test_gmsh2[True-mesh2] PASSED                         [ 16%]
test/test_gmsh.py::test_gmsh2[True-mesh3] PASSED                         [ 16%]
test/test_gmsh.py::test_gmsh2[True-mesh4] PASSED                         [ 16%]
test/test_gmsh.py::test_gmsh2[True-mesh5] PASSED                         [ 16%]
test/test_gmsh.py::test_gmsh2[True-mesh6] PASSED                         [ 17%]
test/test_gmsh.py::test_gmsh2[True-mesh7] PASSED                         [ 17%]
test/test_gmsh.py::test_gmsh2[True-mesh8] PASSED                         [ 17%]
test/test_gmsh.py::test_gmsh2[True-mesh9] PASSED                         [ 17%]
test/test_gmsh.py::test_gmsh2[True-mesh10] PASSED                        [ 17%]
test/test_gmsh.py::test_gmsh2[True-mesh11] PASSED                        [ 18%]
test/test_gmsh.py::test_gmsh2[True-mesh12] PASSED                        [ 18%]
test/test_gmsh.py::test_gmsh2[True-mesh13] PASSED                        [ 18%]
test/test_gmsh.py::test_gmsh2[True-mesh14] PASSED                        [ 18%]
test/test_gmsh.py::test_gmsh2[True-mesh15] PASSED                        [ 19%]
test/test_gmsh.py::test_gmsh2[True-mesh16] PASSED                        [ 19%]
test/test_gmsh.py::test_gmsh2[True-mesh17] PASSED                        [ 19%]
test/test_gmsh.py::test_gmsh2[True-mesh18] PASSED                        [ 19%]
test/test_gmsh.py::test_gmsh40[False-mesh0] PASSED                       [ 19%]
test/test_gmsh.py::test_gmsh40[False-mesh1] PASSED                       [ 20%]
test/test_gmsh.py::test_gmsh40[False-mesh2] PASSED                       [ 20%]
test/test_gmsh.py::test_gmsh40[False-mesh3] PASSED                       [ 20%]
test/test_gmsh.py::test_gmsh40[False-mesh4] PASSED                       [ 20%]
test/test_gmsh.py::test_gmsh40[False-mesh5] PASSED                       [ 21%]
test/test_gmsh.py::test_gmsh40[False-mesh6] PASSED                       [ 21%]
test/test_gmsh.py::test_gmsh40[False-mesh7] PASSED                       [ 21%]
test/test_gmsh.py::test_gmsh40[False-mesh8] PASSED                       [ 21%]
test/test_gmsh.py::test_gmsh40[False-mesh9] PASSED                       [ 21%]
test/test_gmsh.py::test_gmsh40[False-mesh10] PASSED                      [ 22%]
test/test_gmsh.py::test_gmsh40[False-mesh11] PASSED                      [ 22%]
test/test_gmsh.py::test_gmsh40[False-mesh12] PASSED                      [ 22%]
test/test_gmsh.py::test_gmsh40[False-mesh13] PASSED                      [ 22%]
test/test_gmsh.py::test_gmsh40[False-mesh14] PASSED                      [ 23%]
test/test_gmsh.py::test_gmsh40[False-mesh15] PASSED                      [ 23%]
test/test_gmsh.py::test_gmsh40[False-mesh16] PASSED                      [ 23%]
test/test_gmsh.py::test_gmsh40[True-mesh0] PASSED                        [ 23%]
test/test_gmsh.py::test_gmsh40[True-mesh1] PASSED                        [ 23%]
test/test_gmsh.py::test_gmsh40[True-mesh2] PASSED                        [ 24%]
test/test_gmsh.py::test_gmsh40[True-mesh3] PASSED                        [ 24%]
test/test_gmsh.py::test_gmsh40[True-mesh4] PASSED                        [ 24%]
test/test_gmsh.py::test_gmsh40[True-mesh5] PASSED                        [ 24%]
test/test_gmsh.py::test_gmsh40[True-mesh6] PASSED                        [ 25%]
test/test_gmsh.py::test_gmsh40[True-mesh7] PASSED                        [ 25%]
test/test_gmsh.py::test_gmsh40[True-mesh8] PASSED                        [ 25%]
test/test_gmsh.py::test_gmsh40[True-mesh9] PASSED                        [ 25%]
test/test_gmsh.py::test_gmsh40[True-mesh10] PASSED                       [ 26%]
test/test_gmsh.py::test_gmsh40[True-mesh11] PASSED                       [ 26%]
test/test_gmsh.py::test_gmsh40[True-mesh12] PASSED                       [ 26%]
test/test_gmsh.py::test_gmsh40[True-mesh13] PASSED                       [ 26%]
test/test_gmsh.py::test_gmsh40[True-mesh14] PASSED                       [ 26%]
test/test_gmsh.py::test_gmsh40[True-mesh15] PASSED                       [ 27%]
test/test_gmsh.py::test_gmsh40[True-mesh16] PASSED                       [ 27%]
test/test_gmsh.py::test_gmsh41[False-mesh0] PASSED                       [ 27%]
test/test_gmsh.py::test_gmsh41[False-mesh1] PASSED                       [ 27%]
test/test_gmsh.py::test_gmsh41[False-mesh2] PASSED                       [ 28%]
test/test_gmsh.py::test_gmsh41[False-mesh3] PASSED                       [ 28%]
test/test_gmsh.py::test_gmsh41[False-mesh4] PASSED                       [ 28%]
test/test_gmsh.py::test_gmsh41[False-mesh5] PASSED                       [ 28%]
test/test_gmsh.py::test_gmsh41[False-mesh6] PASSED                       [ 28%]
test/test_gmsh.py::test_gmsh41[False-mesh7] PASSED                       [ 29%]
test/test_gmsh.py::test_gmsh41[False-mesh8] PASSED                       [ 29%]
test/test_gmsh.py::test_gmsh41[False-mesh9] PASSED                       [ 29%]
test/test_gmsh.py::test_gmsh41[False-mesh10] PASSED                      [ 29%]
test/test_gmsh.py::test_gmsh41[False-mesh11] PASSED                      [ 30%]
test/test_gmsh.py::test_gmsh41[False-mesh12] PASSED                      [ 30%]
test/test_gmsh.py::test_gmsh41[False-mesh13] PASSED                      [ 30%]
test/test_gmsh.py::test_gmsh41[False-mesh14] PASSED                      [ 30%]
test/test_gmsh.py::test_gmsh41[False-mesh15] PASSED                      [ 30%]
test/test_gmsh.py::test_gmsh41[False-mesh16] PASSED                      [ 31%]
test/test_gmsh.py::test_gmsh41[False-mesh17] PASSED                      [ 31%]
test/test_gmsh.py::test_gmsh41[True-mesh0] PASSED                        [ 31%]
test/test_gmsh.py::test_gmsh41[True-mesh1] PASSED                        [ 31%]
test/test_gmsh.py::test_gmsh41[True-mesh2] PASSED                        [ 32%]
test/test_gmsh.py::test_gmsh41[True-mesh3] PASSED                        [ 32%]
test/test_gmsh.py::test_gmsh41[True-mesh4] PASSED                        [ 32%]
test/test_gmsh.py::test_gmsh41[True-mesh5] PASSED                        [ 32%]
test/test_gmsh.py::test_gmsh41[True-mesh6] PASSED                        [ 32%]
test/test_gmsh.py::test_gmsh41[True-mesh7] PASSED                        [ 33%]
test/test_gmsh.py::test_gmsh41[True-mesh8] PASSED                        [ 33%]
test/test_gmsh.py::test_gmsh41[True-mesh9] PASSED                        [ 33%]
test/test_gmsh.py::test_gmsh41[True-mesh10] PASSED                       [ 33%]
test/test_gmsh.py::test_gmsh41[True-mesh11] PASSED                       [ 34%]
test/test_gmsh.py::test_gmsh41[True-mesh12] PASSED                       [ 34%]
test/test_gmsh.py::test_gmsh41[True-mesh13] PASSED                       [ 34%]
test/test_gmsh.py::test_gmsh41[True-mesh14] PASSED                       [ 34%]
test/test_gmsh.py::test_gmsh41[True-mesh15] PASSED                       [ 34%]
test/test_gmsh.py::test_gmsh41[True-mesh16] PASSED                       [ 35%]
test/test_gmsh.py::test_gmsh41[True-mesh17] PASSED                       [ 35%]
test/test_gmsh.py::test_generic_io PASSED                                [ 35%]
test/test_gmsh.py::test_reference_file[False-insulated-2.2.msh-2.001762136876221-ref_num_cells0] PASSED [ 35%]
test/test_gmsh.py::test_reference_file[True-insulated-2.2.msh-2.001762136876221-ref_num_cells0] PASSED [ 36%]
test/test_gmsh.py::test_reference_file_readonly[insulated-4.1.msh-2.001762136876221-ref_num_cells0] PASSED [ 36%]
test/test_helpers.py::test_read_str PASSED                               [ 36%]
test/test_helpers.py::test_read_pathlike PASSED                          [ 36%]
test/test_helpers.py::test_read_buffer PASSED                            [ 36%]
test/test_helpers.py::test_write_str PASSED                              [ 37%]
test/test_helpers.py::test_write_pathlike PASSED                         [ 37%]
test/test_helpers.py::test_write_buffer PASSED                           [ 37%]
test/test_mdpa.py::test_io[mesh0] PASSED                                 [ 37%]
test/test_mdpa.py::test_io[mesh1] PASSED                                 [ 38%]
test/test_mdpa.py::test_io[mesh2] PASSED                                 [ 38%]
test/test_mdpa.py::test_io[mesh3] PASSED                                 [ 38%]
test/test_mdpa.py::test_io[mesh4] PASSED                                 [ 38%]
test/test_mdpa.py::test_io[mesh5] PASSED                                 [ 39%]
test/test_mdpa.py::test_io[mesh6] PASSED                                 [ 39%]
test/test_mdpa.py::test_io[mesh7] PASSED                                 [ 39%]
test/test_mdpa.py::test_io[mesh8] PASSED                                 [ 39%]
test/test_mdpa.py::test_io[mesh9] PASSED                                 [ 39%]
test/test_mdpa.py::test_generic_io PASSED                                [ 40%]
test/test_med.py::test_io[mesh0] PASSED                                  [ 40%]
test/test_med.py::test_io[mesh1] PASSED                                  [ 40%]
test/test_med.py::test_io[mesh2] PASSED                                  [ 40%]
test/test_med.py::test_io[mesh3] PASSED                                  [ 41%]
test/test_med.py::test_io[mesh4] PASSED                                  [ 41%]
test/test_med.py::test_io[mesh5] PASSED                                  [ 41%]
test/test_med.py::test_io[mesh6] PASSED                                  [ 41%]
test/test_med.py::test_io[mesh7] PASSED                                  [ 41%]
test/test_med.py::test_io[mesh8] PASSED                                  [ 42%]
test/test_med.py::test_io[mesh9] PASSED                                  [ 42%]
test/test_med.py::test_io[mesh10] PASSED                                 [ 42%]
test/test_med.py::test_io[mesh11] PASSED                                 [ 42%]
test/test_med.py::test_io[mesh12] PASSED                                 [ 43%]
test/test_med.py::test_io[mesh13] PASSED                                 [ 43%]
test/test_med.py::test_io[mesh14] PASSED                                 [ 43%]
test/test_med.py::test_io[mesh15] PASSED                                 [ 43%]
test/test_med.py::test_io[mesh16] PASSED                                 [ 43%]
test/test_med.py::test_io[mesh17] PASSED                                 [ 44%]
test/test_med.py::test_generic_io PASSED                                 [ 44%]
test/test_med.py::test_reference_file_with_mixed_cells PASSED            [ 44%]
test/test_med.py::test_reference_file_with_point_cell_data PASSED        [ 44%]
test/test_medit.py::test_io[mesh0] PASSED                                [ 45%]
test/test_medit.py::test_io[mesh1] PASSED                                [ 45%]
test/test_medit.py::test_io[mesh2] PASSED                                [ 45%]
test/test_medit.py::test_io[mesh3] PASSED                                [ 45%]
test/test_medit.py::test_io[mesh4] PASSED                                [ 45%]
test/test_medit.py::test_io[mesh5] PASSED                                [ 46%]
test/test_medit.py::test_io[mesh6] PASSED                                [ 46%]
test/test_medit.py::test_generic_io PASSED                               [ 46%]
test/test_mesh.py::test PASSED                                           [ 46%]
test/test_moab.py::test_io[mesh0] PASSED                                 [ 47%]
test/test_moab.py::test_io[mesh1] PASSED                                 [ 47%]
test/test_moab.py::test_io[mesh2] PASSED                                 [ 47%]
test/test_moab.py::test_io[mesh3] PASSED                                 [ 47%]
test/test_moab.py::test_generic_io PASSED                                [ 47%]
test/test_nastran.py::test[mesh0] PASSED                                 [ 48%]
test/test_nastran.py::test[mesh1] PASSED                                 [ 48%]
test/test_nastran.py::test[mesh2] PASSED                                 [ 48%]
test/test_nastran.py::test[mesh3] PASSED                                 [ 48%]
test/test_nastran.py::test[mesh4] PASSED                                 [ 49%]
test/test_nastran.py::test[mesh5] PASSED                                 [ 49%]
test/test_nastran.py::test[mesh6] PASSED                                 [ 49%]
test/test_nastran.py::test[mesh7] PASSED                                 [ 49%]
test/test_nastran.py::test[mesh8] PASSED                                 [ 50%]
test/test_nastran.py::test[mesh9] PASSED                                 [ 50%]
test/test_nastran.py::test_reference_file PASSED                         [ 50%]
test/test_neuroglancer.py::test_neuroglancer[mesh0] PASSED               [ 50%]
test/test_neuroglancer.py::test_reference_file[simple1-20-4] PASSED      [ 50%]
test/test_obj.py::test_obj[mesh0] PASSED                                 [ 51%]
test/test_obj.py::test_obj[mesh1] PASSED                                 [ 51%]
test/test_obj.py::test_obj[mesh2] PASSED                                 [ 51%]
test/test_obj.py::test_reference_file[elephav.obj-367837.217245-1148] PASSED [ 51%]
test/test_off.py::test_io[mesh0] PASSED                                  [ 52%]
test/test_off.py::test_generic_io PASSED                                 [ 52%]
test/test_permas.py::test_io[mesh0] PASSED                               [ 52%]
test/test_permas.py::test_io[mesh1] PASSED                               [ 52%]
test/test_permas.py::test_io[mesh2] PASSED                               [ 52%]
test/test_permas.py::test_io[mesh3] PASSED                               [ 53%]
test/test_permas.py::test_io[mesh4] PASSED                               [ 53%]
test/test_permas.py::test_io[mesh5] PASSED                               [ 53%]
test/test_permas.py::test_io[mesh6] PASSED                               [ 53%]
test/test_permas.py::test_io[mesh7] PASSED                               [ 54%]
test/test_permas.py::test_generic_io PASSED                              [ 54%]
test/test_ply.py::test_ply[False-mesh0] FAILED                           [ 54%]
test/test_ply.py::test_ply[False-mesh1] PASSED                           [ 54%]
test/test_ply.py::test_ply[True-mesh0] FAILED                            [ 54%]
test/test_ply.py::test_ply[True-mesh1] PASSED                            [ 55%]
test/test_ply.py::test_reference_file[bun_zipper_res4.ply-34.14583969116211-948] PASSED [ 55%]
test/test_stl.py::test_stl[False-1e-15-mesh0] PASSED                     [ 55%]
test/test_stl.py::test_stl[True-1e-08-mesh0] PASSED                      [ 55%]
test/test_svg.py::test[mesh0] PASSED                                     [ 56%]
test/test_svg.py::test[mesh1] PASSED                                     [ 56%]
test/test_svg.py::test[mesh2] PASSED                                     [ 56%]
test/test_svg.py::test[mesh3] PASSED                                     [ 56%]
test/test_tetgen.py::test[mesh0] PASSED                                  [ 56%]
test/test_ugrid.py::test_io[1e-07-.ugrid-mesh0] PASSED                   [ 57%]
test/test_ugrid.py::test_io[1e-07-.ugrid-mesh1] PASSED                   [ 57%]
test/test_ugrid.py::test_io[1e-07-.ugrid-mesh2] PASSED                   [ 57%]
test/test_ugrid.py::test_io[1e-07-.ugrid-mesh3] PASSED                   [ 57%]
test/test_ugrid.py::test_io[1e-07-.ugrid-mesh4] PASSED                   [ 58%]
test/test_ugrid.py::test_io[1e-15-.b8.ugrid-mesh0] PASSED                [ 58%]
test/test_ugrid.py::test_io[1e-15-.b8.ugrid-mesh1] PASSED                [ 58%]
test/test_ugrid.py::test_io[1e-15-.b8.ugrid-mesh2] PASSED                [ 58%]
test/test_ugrid.py::test_io[1e-15-.b8.ugrid-mesh3] PASSED                [ 58%]
test/test_ugrid.py::test_io[1e-15-.b8.ugrid-mesh4] PASSED                [ 59%]
test/test_ugrid.py::test_io[1e-07-.b4.ugrid-mesh0] PASSED                [ 59%]
test/test_ugrid.py::test_io[1e-07-.b4.ugrid-mesh1] PASSED                [ 59%]
test/test_ugrid.py::test_io[1e-07-.b4.ugrid-mesh2] PASSED                [ 59%]
test/test_ugrid.py::test_io[1e-07-.b4.ugrid-mesh3] PASSED                [ 60%]
test/test_ugrid.py::test_io[1e-07-.b4.ugrid-mesh4] PASSED                [ 60%]
test/test_ugrid.py::test_io[1e-15-.lb8.ugrid-mesh0] PASSED               [ 60%]
test/test_ugrid.py::test_io[1e-15-.lb8.ugrid-mesh1] PASSED               [ 60%]
test/test_ugrid.py::test_io[1e-15-.lb8.ugrid-mesh2] PASSED               [ 60%]
test/test_ugrid.py::test_io[1e-15-.lb8.ugrid-mesh3] PASSED               [ 61%]
test/test_ugrid.py::test_io[1e-15-.lb8.ugrid-mesh4] PASSED               [ 61%]
test/test_ugrid.py::test_io[1e-07-.lb4.ugrid-mesh0] PASSED               [ 61%]
test/test_ugrid.py::test_io[1e-07-.lb4.ugrid-mesh1] PASSED               [ 61%]
test/test_ugrid.py::test_io[1e-07-.lb4.ugrid-mesh2] PASSED               [ 62%]
test/test_ugrid.py::test_io[1e-07-.lb4.ugrid-mesh3] PASSED               [ 62%]
test/test_ugrid.py::test_io[1e-07-.lb4.ugrid-mesh4] PASSED               [ 62%]
test/test_ugrid.py::test_io[1e-15-.r8.ugrid-mesh0] PASSED                [ 62%]
test/test_ugrid.py::test_io[1e-15-.r8.ugrid-mesh1] PASSED                [ 63%]
test/test_ugrid.py::test_io[1e-15-.r8.ugrid-mesh2] PASSED                [ 63%]
test/test_ugrid.py::test_io[1e-15-.r8.ugrid-mesh3] PASSED                [ 63%]
test/test_ugrid.py::test_io[1e-15-.r8.ugrid-mesh4] PASSED                [ 63%]
test/test_ugrid.py::test_io[1e-07-.r4.ugrid-mesh0] PASSED                [ 63%]
test/test_ugrid.py::test_io[1e-07-.r4.ugrid-mesh1] PASSED                [ 64%]
test/test_ugrid.py::test_io[1e-07-.r4.ugrid-mesh2] PASSED                [ 64%]
test/test_ugrid.py::test_io[1e-07-.r4.ugrid-mesh3] PASSED                [ 64%]
test/test_ugrid.py::test_io[1e-07-.r4.ugrid-mesh4] PASSED                [ 64%]
test/test_ugrid.py::test_io[1e-15-.lr8.ugrid-mesh0] PASSED               [ 65%]
test/test_ugrid.py::test_io[1e-15-.lr8.ugrid-mesh1] PASSED               [ 65%]
test/test_ugrid.py::test_io[1e-15-.lr8.ugrid-mesh2] PASSED               [ 65%]
test/test_ugrid.py::test_io[1e-15-.lr8.ugrid-mesh3] PASSED               [ 65%]
test/test_ugrid.py::test_io[1e-15-.lr8.ugrid-mesh4] PASSED               [ 65%]
test/test_ugrid.py::test_io[1e-07-.lr4.ugrid-mesh0] PASSED               [ 66%]
test/test_ugrid.py::test_io[1e-07-.lr4.ugrid-mesh1] PASSED               [ 66%]
test/test_ugrid.py::test_io[1e-07-.lr4.ugrid-mesh2] PASSED               [ 66%]
test/test_ugrid.py::test_io[1e-07-.lr4.ugrid-mesh3] PASSED               [ 66%]
test/test_ugrid.py::test_io[1e-07-.lr4.ugrid-mesh4] PASSED               [ 67%]
test/test_ugrid.py::test_generic_io PASSED                               [ 67%]
test/test_ugrid.py::test_reference_file[sphere_mixed.1.lb8.ugrid-3270-864-0-3024-9072-0-ref_tag_counts0] PASSED [ 67%]
test/test_ugrid.py::test_reference_file[hch_strct.4.lb8.ugrid-306-12-178-96-0-144-ref_tag_counts1] PASSED [ 67%]
test/test_ugrid.py::test_volume[pyra_cube.ugrid-1.0-1e-15] PASSED        [ 67%]
test/test_vtk.py::test[True-mesh0] PASSED                                [ 68%]
test/test_vtk.py::test[True-mesh1] PASSED                                [ 68%]
test/test_vtk.py::test[True-mesh2] PASSED                                [ 68%]
test/test_vtk.py::test[True-mesh3] PASSED                                [ 68%]
test/test_vtk.py::test[True-mesh4] PASSED                                [ 69%]
test/test_vtk.py::test[True-mesh5] PASSED                                [ 69%]
test/test_vtk.py::test[True-mesh6] PASSED                                [ 69%]
test/test_vtk.py::test[True-mesh7] PASSED                                [ 69%]
test/test_vtk.py::test[True-mesh8] PASSED                                [ 69%]
test/test_vtk.py::test[True-mesh9] PASSED                                [ 70%]
test/test_vtk.py::test[True-mesh10] PASSED                               [ 70%]
test/test_vtk.py::test[True-mesh11] PASSED                               [ 70%]
test/test_vtk.py::test[True-mesh12] PASSED                               [ 70%]
test/test_vtk.py::test[True-mesh13] PASSED                               [ 71%]
test/test_vtk.py::test[True-mesh14] PASSED                               [ 71%]
test/test_vtk.py::test[True-mesh15] PASSED                               [ 71%]
test/test_vtk.py::test[True-mesh16] PASSED                               [ 71%]
test/test_vtk.py::test[True-mesh17] PASSED                               [ 71%]
test/test_vtk.py::test[True-mesh18] PASSED                               [ 72%]
test/test_vtk.py::test[False-mesh0] PASSED                               [ 72%]
test/test_vtk.py::test[False-mesh1] PASSED                               [ 72%]
test/test_vtk.py::test[False-mesh2] PASSED                               [ 72%]
test/test_vtk.py::test[False-mesh3] PASSED                               [ 73%]
test/test_vtk.py::test[False-mesh4] PASSED                               [ 73%]
test/test_vtk.py::test[False-mesh5] PASSED                               [ 73%]
test/test_vtk.py::test[False-mesh6] PASSED                               [ 73%]
test/test_vtk.py::test[False-mesh7] PASSED                               [ 73%]
test/test_vtk.py::test[False-mesh8] PASSED                               [ 74%]
test/test_vtk.py::test[False-mesh9] PASSED                               [ 74%]
test/test_vtk.py::test[False-mesh10] PASSED                              [ 74%]
test/test_vtk.py::test[False-mesh11] PASSED                              [ 74%]
test/test_vtk.py::test[False-mesh12] PASSED                              [ 75%]
test/test_vtk.py::test[False-mesh13] PASSED                              [ 75%]
test/test_vtk.py::test[False-mesh14] PASSED                              [ 75%]
test/test_vtk.py::test[False-mesh15] PASSED                              [ 75%]
test/test_vtk.py::test[False-mesh16] PASSED                              [ 76%]
test/test_vtk.py::test[False-mesh17] PASSED                              [ 76%]
test/test_vtk.py::test[False-mesh18] PASSED                              [ 76%]
test/test_vtk.py::test_generic_io PASSED                                 [ 76%]
test/test_vtk.py::test_reference_file[False-rbc_001.vtk-0.00031280518-996] PASSED [ 76%]
test/test_vtk.py::test_reference_file[True-rbc_001.vtk-0.00031280518-996] PASSED [ 77%]
test/test_vtk.py::test_structured[00_image.vtk-quad-81-100] PASSED       [ 77%]
test/test_vtk.py::test_structured[01_image.vtk-hexahedron-72-147] PASSED [ 77%]
test/test_vtk.py::test_structured[02_structured.vtk-hexahedron-72-147] PASSED [ 77%]
test/test_vtk.py::test_structured[03_rectilinear.vtk-hexahedron-72-147] PASSED [ 78%]
test/test_vtk.py::test_structured[04_rectilinear.vtk-quad-27-40] PASSED  [ 78%]
test/test_vtk.py::test_structured[05_rectilinear.vtk-quad-27-40] PASSED  [ 78%]
test/test_vtk.py::test_pathlike PASSED                                   [ 78%]
test/test_vtu.py::test[False-mesh0] PASSED                               [ 78%]
test/test_vtu.py::test[False-mesh1] PASSED                               [ 79%]
test/test_vtu.py::test[False-mesh2] PASSED                               [ 79%]
test/test_vtu.py::test[False-mesh3] PASSED                               [ 79%]
test/test_vtu.py::test[False-mesh4] PASSED                               [ 79%]
test/test_vtu.py::test[False-mesh5] PASSED                               [ 80%]
test/test_vtu.py::test[False-mesh6] PASSED                               [ 80%]
test/test_vtu.py::test[False-mesh7] PASSED                               [ 80%]
test/test_vtu.py::test[False-mesh8] PASSED                               [ 80%]
test/test_vtu.py::test[False-mesh9] PASSED                               [ 80%]
test/test_vtu.py::test[False-mesh10] PASSED                              [ 81%]
test/test_vtu.py::test[False-mesh11] PASSED                              [ 81%]
test/test_vtu.py::test[False-mesh12] PASSED                              [ 81%]
test/test_vtu.py::test[False-mesh13] PASSED                              [ 81%]
test/test_vtu.py::test[False-mesh14] PASSED                              [ 82%]
test/test_vtu.py::test[False-mesh15] PASSED                              [ 82%]
test/test_vtu.py::test[True-mesh0] PASSED                                [ 82%]
test/test_vtu.py::test[True-mesh1] PASSED                                [ 82%]
test/test_vtu.py::test[True-mesh2] PASSED                                [ 82%]
test/test_vtu.py::test[True-mesh3] PASSED                                [ 83%]
test/test_vtu.py::test[True-mesh4] PASSED                                [ 83%]
test/test_vtu.py::test[True-mesh5] PASSED                                [ 83%]
test/test_vtu.py::test[True-mesh6] PASSED                                [ 83%]
test/test_vtu.py::test[True-mesh7] PASSED                                [ 84%]
test/test_vtu.py::test[True-mesh8] PASSED                                [ 84%]
test/test_vtu.py::test[True-mesh9] PASSED                                [ 84%]
test/test_vtu.py::test[True-mesh10] PASSED                               [ 84%]
test/test_vtu.py::test[True-mesh11] PASSED                               [ 84%]
test/test_vtu.py::test[True-mesh12] PASSED                               [ 85%]
test/test_vtu.py::test[True-mesh13] PASSED                               [ 85%]
test/test_vtu.py::test[True-mesh14] PASSED                               [ 85%]
test/test_vtu.py::test[True-mesh15] PASSED                               [ 85%]
test/test_vtu.py::test_generic_io PASSED                                 [ 86%]
test/test_wkt.py::test_wkt[mesh0] PASSED                                 [ 86%]
test/test_wkt.py::test_reference_file[simple.wkt-4-2] PASSED             [ 86%]
test/test_wkt.py::test_reference_file[whitespaced.wkt-3.2-2] PASSED      [ 86%]
test/test_xdmf.py::test_xdmf3[XML-mesh0] PASSED                          [ 86%]
test/test_xdmf.py::test_xdmf3[XML-mesh1] PASSED                          [ 87%]
test/test_xdmf.py::test_xdmf3[XML-mesh2] PASSED                          [ 87%]
test/test_xdmf.py::test_xdmf3[XML-mesh3] PASSED                          [ 87%]
test/test_xdmf.py::test_xdmf3[XML-mesh4] PASSED                          [ 87%]
test/test_xdmf.py::test_xdmf3[XML-mesh5] PASSED                          [ 88%]
test/test_xdmf.py::test_xdmf3[XML-mesh6] PASSED                          [ 88%]
test/test_xdmf.py::test_xdmf3[XML-mesh7] PASSED                          [ 88%]
test/test_xdmf.py::test_xdmf3[XML-mesh8] PASSED                          [ 88%]
test/test_xdmf.py::test_xdmf3[XML-mesh9] PASSED                          [ 89%]
test/test_xdmf.py::test_xdmf3[XML-mesh10] PASSED                         [ 89%]
test/test_xdmf.py::test_xdmf3[XML-mesh11] PASSED                         [ 89%]
test/test_xdmf.py::test_xdmf3[XML-mesh12] PASSED                         [ 89%]
test/test_xdmf.py::test_xdmf3[XML-mesh13] PASSED                         [ 89%]
test/test_xdmf.py::test_xdmf3[Binary-mesh0] PASSED                       [ 90%]
test/test_xdmf.py::test_xdmf3[Binary-mesh1] PASSED                       [ 90%]
test/test_xdmf.py::test_xdmf3[Binary-mesh2] PASSED                       [ 90%]
test/test_xdmf.py::test_xdmf3[Binary-mesh3] PASSED                       [ 90%]
test/test_xdmf.py::test_xdmf3[Binary-mesh4] PASSED                       [ 91%]
test/test_xdmf.py::test_xdmf3[Binary-mesh5] PASSED                       [ 91%]
test/test_xdmf.py::test_xdmf3[Binary-mesh6] PASSED                       [ 91%]
test/test_xdmf.py::test_xdmf3[Binary-mesh7] PASSED                       [ 91%]
test/test_xdmf.py::test_xdmf3[Binary-mesh8] PASSED                       [ 91%]
test/test_xdmf.py::test_xdmf3[Binary-mesh9] PASSED                       [ 92%]
test/test_xdmf.py::test_xdmf3[Binary-mesh10] PASSED                      [ 92%]
test/test_xdmf.py::test_xdmf3[Binary-mesh11] PASSED                      [ 92%]
test/test_xdmf.py::test_xdmf3[Binary-mesh12] PASSED                      [ 92%]
test/test_xdmf.py::test_xdmf3[Binary-mesh13] PASSED                      [ 93%]
test/test_xdmf.py::test_xdmf3[HDF-mesh0] PASSED                          [ 93%]
test/test_xdmf.py::test_xdmf3[HDF-mesh1] PASSED                          [ 93%]
test/test_xdmf.py::test_xdmf3[HDF-mesh2] PASSED                          [ 93%]
test/test_xdmf.py::test_xdmf3[HDF-mesh3] PASSED                          [ 93%]
test/test_xdmf.py::test_xdmf3[HDF-mesh4] PASSED                          [ 94%]
test/test_xdmf.py::test_xdmf3[HDF-mesh5] PASSED                          [ 94%]
test/test_xdmf.py::test_xdmf3[HDF-mesh6] PASSED                          [ 94%]
test/test_xdmf.py::test_xdmf3[HDF-mesh7] PASSED                          [ 94%]
test/test_xdmf.py::test_xdmf3[HDF-mesh8] PASSED                          [ 95%]
test/test_xdmf.py::test_xdmf3[HDF-mesh9] PASSED                          [ 95%]
test/test_xdmf.py::test_xdmf3[HDF-mesh10] PASSED                         [ 95%]
test/test_xdmf.py::test_xdmf3[HDF-mesh11] PASSED                         [ 95%]
test/test_xdmf.py::test_xdmf3[HDF-mesh12] PASSED                         [ 95%]
test/test_xdmf.py::test_xdmf3[HDF-mesh13] PASSED                         [ 96%]
test/test_xdmf.py::test_compression[mesh0] PASSED                        [ 96%]
test/test_xdmf.py::test_compression[mesh1] PASSED                        [ 96%]
test/test_xdmf.py::test_compression[mesh2] PASSED                        [ 96%]
test/test_xdmf.py::test_compression[mesh3] PASSED                        [ 97%]
test/test_xdmf.py::test_compression[mesh4] PASSED                        [ 97%]
test/test_xdmf.py::test_compression[mesh5] PASSED                        [ 97%]
test/test_xdmf.py::test_compression[mesh6] PASSED                        [ 97%]
test/test_xdmf.py::test_compression[mesh7] PASSED                        [ 97%]
test/test_xdmf.py::test_compression[mesh8] PASSED                        [ 98%]
test/test_xdmf.py::test_compression[mesh9] PASSED                        [ 98%]
test/test_xdmf.py::test_compression[mesh10] PASSED                       [ 98%]
test/test_xdmf.py::test_compression[mesh11] PASSED                       [ 98%]
test/test_xdmf.py::test_compression[mesh12] PASSED                       [ 99%]
test/test_xdmf.py::test_compression[mesh13] PASSED                       [ 99%]
test/test_xdmf.py::test_generic_io PASSED                                [ 99%]
test/test_xdmf.py::test_time_series PASSED                               [ 99%]
test/test_xdmf.py::test_information_xdmf PASSED                          [100%]

=================================== FAILURES ===================================
____________________________ test_ply[False-mesh0] _____________________________

mesh = <meshio mesh object>
  Number of points: 4
  Number of cells:
    triangle: 2
  Point data: a, b
binary = False

    @pytest.mark.parametrize(
        "mesh",
        [
            # helpers.tri_mesh,
            # helpers.quad_mesh,
            # helpers.tri_quad_mesh,
            helpers.add_point_data(helpers.tri_mesh, 1, dtype=int),
            helpers.add_point_data(helpers.tri_mesh, 1, dtype=float),
            # helpers.add_cell_data(helpers.tri_mesh, 1),
            # helpers.add_cell_data(helpers.tri_mesh, 3),
            # helpers.add_cell_data(helpers.tri_mesh, 9),
        ],
    )
    @pytest.mark.parametrize("binary", [False, True])
    def test_ply(mesh, binary):
        def writer(*args, **kwargs):
            return meshio.ply.write(*args, binary=binary, **kwargs)
    
        for key in mesh.cells:
            mesh.cells[key] = mesh.cells[key].astype(numpy.int32)
    
>       helpers.write_read(writer, meshio.ply.read, mesh, 1.0e-12)

test/test_ply.py:31: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
test/helpers.py:269: in write_read
    writer(filepath, input_mesh)
test/test_ply.py:26: in writer
    return meshio.ply.write(*args, binary=binary, **kwargs)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

filename = '/tmp/tmpu8q19dyh/test.dat'
mesh = <meshio mesh object>
  Number of points: 4
  Number of cells:
    triangle: 2
  Point data: a, b
binary = False

    def write(filename, mesh, binary=True):  # noqa: C901
        for key in mesh.cells:
            if key not in [
                "triangle",
                "quad",
            ]:
                raise WriteError("Can only deal with triangular and quadrilateral faces")
    
        with open_file(filename, "wb") as fh:
            fh.write(b"ply\n")
            fh.write(b"comment Created by meshio\n")
    
            if binary:
                fh.write(
                    "format binary_{}_endian 1.0\n".format(sys.byteorder).encode("utf-8")
                )
            else:
                fh.write(b"format ascii 1.0\n")
    
            # counts
            fh.write("element vertex {:d}\n".format(mesh.points.shape[0]).encode("utf-8"))
            #
            dim_names = ["x", "y", "z"]
            type_name_table = {
                numpy.dtype(numpy.int8): "int8",
                numpy.dtype(numpy.int64): "int64",
                numpy.dtype(numpy.float32): "float",
                numpy.dtype(numpy.float64): "double",
            }
            for k in range(mesh.points.shape[1]):
                type_name = type_name_table[mesh.points.dtype]
                fh.write("property {} {}\n".format(type_name, dim_names[k]).encode("utf-8"))
            for key, value in mesh.point_data.items():
>               type_name = type_name_table[value.dtype]
E               KeyError: dtype('int32')

/usr/lib/python3/dist-packages/meshio/ply/_ply.py:317: KeyError
_____________________________ test_ply[True-mesh0] _____________________________

mesh = <meshio mesh object>
  Number of points: 4
  Number of cells:
    triangle: 2
  Point data: a, b
binary = True

    @pytest.mark.parametrize(
        "mesh",
        [
            # helpers.tri_mesh,
            # helpers.quad_mesh,
            # helpers.tri_quad_mesh,
            helpers.add_point_data(helpers.tri_mesh, 1, dtype=int),
            helpers.add_point_data(helpers.tri_mesh, 1, dtype=float),
            # helpers.add_cell_data(helpers.tri_mesh, 1),
            # helpers.add_cell_data(helpers.tri_mesh, 3),
            # helpers.add_cell_data(helpers.tri_mesh, 9),
        ],
    )
    @pytest.mark.parametrize("binary", [False, True])
    def test_ply(mesh, binary):
        def writer(*args, **kwargs):
            return meshio.ply.write(*args, binary=binary, **kwargs)
    
        for key in mesh.cells:
            mesh.cells[key] = mesh.cells[key].astype(numpy.int32)
    
>       helpers.write_read(writer, meshio.ply.read, mesh, 1.0e-12)

test/test_ply.py:31: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
test/helpers.py:269: in write_read
    writer(filepath, input_mesh)
test/test_ply.py:26: in writer
    return meshio.ply.write(*args, binary=binary, **kwargs)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

filename = '/tmp/tmpcv8hoee1/test.dat'
mesh = <meshio mesh object>
  Number of points: 4
  Number of cells:
    triangle: 2
  Point data: a, b
binary = True

    def write(filename, mesh, binary=True):  # noqa: C901
        for key in mesh.cells:
            if key not in [
                "triangle",
                "quad",
            ]:
                raise WriteError("Can only deal with triangular and quadrilateral faces")
    
        with open_file(filename, "wb") as fh:
            fh.write(b"ply\n")
            fh.write(b"comment Created by meshio\n")
    
            if binary:
                fh.write(
                    "format binary_{}_endian 1.0\n".format(sys.byteorder).encode("utf-8")
                )
            else:
                fh.write(b"format ascii 1.0\n")
    
            # counts
            fh.write("element vertex {:d}\n".format(mesh.points.shape[0]).encode("utf-8"))
            #
            dim_names = ["x", "y", "z"]
            type_name_table = {
                numpy.dtype(numpy.int8): "int8",
                numpy.dtype(numpy.int64): "int64",
                numpy.dtype(numpy.float32): "float",
                numpy.dtype(numpy.float64): "double",
            }
            for k in range(mesh.points.shape[1]):
                type_name = type_name_table[mesh.points.dtype]
                fh.write("property {} {}\n".format(type_name, dim_names[k]).encode("utf-8"))
            for key, value in mesh.point_data.items():
>               type_name = type_name_table[value.dtype]
E               KeyError: dtype('int32')

/usr/lib/python3/dist-packages/meshio/ply/_ply.py:317: KeyError
===================== 2 failed, 444 passed in 8.01 seconds =====================
autopkgtest [10:03:29]: test command1: -----------------------]
autopkgtest [10:03:31]: test command1:  - - - - - - - - - - results - - - - - - - - - -
command1             FAIL non-zero exit status 1
autopkgtest [10:03:33]: @@@@@@@@@@@@@@@@@@@@ summary
command1             FAIL non-zero exit status 1
```

any idea?
http://autopkgtest.ubuntu.com/packages/p/python-meshio/focal/armhf

--- a/README.md
+++ b/README.md
@@ -16,7 +16,7 @@
 There are various mesh formats available for representing unstructured meshes.
 meshio can read and write all of the following and smoothly converts between them:
 
- [Abaqus](http://abaqus.software.polimi.it/v6.14/index.html),
+> [Abaqus](http://abaqus.software.polimi.it/v6.14/index.html),
  [ANSYS msh](https://www.afs.enea.it/fluent/Public/Fluent-Doc/PDF/chp03.pdf),
  [CGNS](https://cgns.github.io/),
  [DOLFIN XML](https://manpages.ubuntu.com/manpages/disco/man1/dolfin-convert.1.html),
@@ -30,7 +30,7 @@ meshio can read and write all of the following and smoothly converts between the
  [Neuroglancer precomputed format](https://github.com/google/neuroglancer/tree/master/src/neuroglancer/datasource/precomputed#mesh-representation-of-segmented-object-surfaces),
  [Gmsh](http://gmsh.info/doc/texinfo/gmsh.html#File-formats) (versions 2 and 4),
  [OBJ](https://en.wikipedia.org/wiki/Wavefront_.obj_file),
- [OFF](https://segeval.cs.princeton.edu/public/off_format.html)m
+ [OFF](https://segeval.cs.princeton.edu/public/off_format.html),
  [PERMAS](https://www.intes.de),
  [PLY](https://en.wikipedia.org/wiki/PLY_(file_format)),
  [STL](https://en.wikipedia.org/wiki/STL_(file_format)),

--- a/README.md
+++ b/README.md
@@ -16,31 +16,31 @@
 There are various mesh formats available for representing unstructured meshes.
 meshio can read and write all of the following and smoothly converts between them:
 
- * [Abaqus](http://abaqus.software.polimi.it/v6.14/index.html)
- * [ANSYS msh](https://www.afs.enea.it/fluent/Public/Fluent-Doc/PDF/chp03.pdf)
- * [CGNS](https://cgns.github.io/)
- * [DOLFIN XML](https://manpages.ubuntu.com/manpages/disco/man1/dolfin-convert.1.html)
- * [Exodus](https://cubit.sandia.gov/public/13.2/help_manual/WebHelp/finite_element_model/exodus/block_specification.htm)
- * [FLAC3D](https://www.itascacg.com/software/flac3d)
- * [H5M](https://www.mcs.anl.gov/~fathom/moab-docs/h5mmain.html)
- * [Kratos/MDPA](https://github.com/KratosMultiphysics/Kratos/wiki/Input-data)
- * [Medit](https://people.sc.fsu.edu/~jburkardt/data/medit/medit.html)
- * [MED/Salome](https://docs.salome-platform.org/latest/dev/MEDCoupling/developer/med-file.html)
- * [Nastran](https://help.autodesk.com/view/NSTRN/2019/ENU/?guid=GUID-42B54ACB-FBE3-47CA-B8FE-475E7AD91A00) (bulk data)
- * [Neuroglancer precomputed format](https://github.com/google/neuroglancer/tree/master/src/neuroglancer/datasource/precomputed#mesh-representation-of-segmented-object-surfaces)
- * [Gmsh](http://gmsh.info/doc/texinfo/gmsh.html#File-formats) (versions 2 and 4)
- * [OBJ](https://en.wikipedia.org/wiki/Wavefront_.obj_file)
- * [OFF](https://segeval.cs.princeton.edu/public/off_format.html)
- * [PERMAS](https://www.intes.de)
- * [PLY](https://en.wikipedia.org/wiki/PLY_(file_format))
- * [STL](https://en.wikipedia.org/wiki/STL_(file_format))
- * [TetGen .node/.ele](https://wias-berlin.de/software/tetgen/fformats.html)
- * [SVG](https://www.w3.org/TR/SVG/) (2D only, output only)
- * [UGRID](http://www.simcenter.msstate.edu/software/downloads/doc/ug_io/3d_grid_file_type_ugrid.html)
- * [VTK](https://www.vtk.org/wp-content/uploads/2015/04/file-formats.pdf)
- * [VTU](https://www.vtk.org/Wiki/VTK_XML_Formats) ([not raw binary data](https://github.com/nschloe/meshio/wiki/VTU-with-raw-binary-data))
- * [WKT](https://en.wikipedia.org/wiki/Well-known_text_representation_of_geometry) ([TIN](https://en.wikipedia.org/wiki/Triangulated_irregular_network))
- * [XDMF](http://www.xdmf.org/index.php/XDMF_Model_and_Format)
+ [Abaqus](http://abaqus.software.polimi.it/v6.14/index.html),
+ [ANSYS msh](https://www.afs.enea.it/fluent/Public/Fluent-Doc/PDF/chp03.pdf),
+ [CGNS](https://cgns.github.io/),
+ [DOLFIN XML](https://manpages.ubuntu.com/manpages/disco/man1/dolfin-convert.1.html),
+ [Exodus](https://cubit.sandia.gov/public/13.2/help_manual/WebHelp/finite_element_model/exodus/block_specification.htm),
+ [FLAC3D](https://www.itascacg.com/software/flac3d),
+ [H5M](https://www.mcs.anl.gov/~fathom/moab-docs/h5mmain.html),
+ [Kratos/MDPA](https://github.com/KratosMultiphysics/Kratos/wiki/Input-data),
+ [Medit](https://people.sc.fsu.edu/~jburkardt/data/medit/medit.html),
+ [MED/Salome](https://docs.salome-platform.org/latest/dev/MEDCoupling/developer/med-file.html),
+ [Nastran](https://help.autodesk.com/view/NSTRN/2019/ENU/?guid=GUID-42B54ACB-FBE3-47CA-B8FE-475E7AD91A00) (bulk data),
+ [Neuroglancer precomputed format](https://github.com/google/neuroglancer/tree/master/src/neuroglancer/datasource/precomputed#mesh-representation-of-segmented-object-surfaces),
+ [Gmsh](http://gmsh.info/doc/texinfo/gmsh.html#File-formats) (versions 2 and 4),
+ [OBJ](https://en.wikipedia.org/wiki/Wavefront_.obj_file),
+ [OFF](https://segeval.cs.princeton.edu/public/off_format.html)m
+ [PERMAS](https://www.intes.de),
+ [PLY](https://en.wikipedia.org/wiki/PLY_(file_format)),
+ [STL](https://en.wikipedia.org/wiki/STL_(file_format)),
+ [TetGen .node/.ele](https://wias-berlin.de/software/tetgen/fformats.html),
+ [SVG](https://www.w3.org/TR/SVG/) (2D only, output only),
+ [UGRID](http://www.simcenter.msstate.edu/software/downloads/doc/ug_io/3d_grid_file_type_ugrid.html),
+ [VTK](https://www.vtk.org/wp-content/uploads/2015/04/file-formats.pdf),
+ [VTU](https://www.vtk.org/Wiki/VTK_XML_Formats) ([not raw binary data](https://github.com/nschloe/meshio/wiki/VTU-with-raw-binary-data)),
+ [WKT](https://en.wikipedia.org/wiki/Well-known_text_representation_of_geometry) ([TIN](https://en.wikipedia.org/wiki/Triangulated_irregular_network)),
+ [XDMF](http://www.xdmf.org/index.php/XDMF_Model_and_Format).
 
 Install with
 ```

--- a/meshio/ply/_ply.py
+++ b/meshio/ply/_ply.py
@@ -304,9 +304,21 @@ def write(filename, mesh, binary=True):  # noqa: C901
         fh.write("element vertex {:d}\n".format(mesh.points.shape[0]).encode("utf-8"))
         #
         dim_names = ["x", "y", "z"]
+        # From <https://en.wikipedia.org/wiki/PLY_(file_format)>:
+        #
+        # > The type can be specified with one of char uchar short ushort int uint float
+        # > double, or one of int8 uint8 int16 uint16 int32 uint32 float32 float64.
+        #
+        # We're adding [u]int64 here.
         type_name_table = {
             numpy.dtype(numpy.int8): "int8",
+            numpy.dtype(numpy.int16): "int16",
+            numpy.dtype(numpy.int32): "int32",
             numpy.dtype(numpy.int64): "int64",
+            numpy.dtype(numpy.uint8): "uint8",
+            numpy.dtype(numpy.uint16): "uint16",
+            numpy.dtype(numpy.uint32): "uint32",
+            numpy.dtype(numpy.uint64): "uint64",
             numpy.dtype(numpy.float32): "float",
             numpy.dtype(numpy.float64): "double",
         }

